onnx/defs/math/defs.cc

```diff
@@ -1897,6 +1897,12 @@ ONNX_OPERATOR_SET_SCHEMA(
             "If given, it has to be a tensor of size C. Otherwise, it is treated as if having all ones.",
             "T",
             OpSchema::Optional)
+        .Input(
```

Comment:
why not make it an attribute?

Reply:
Talked with @codemzs offline, he is ok to make this an attribute for NllLoss and SoftmaxCrossEntropy.
cc : @SherlockNoMad